React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,11 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if ([UIDevice currentDevice].systemVersion.floatValue >= 9) {
```

Comment:
Ok can you just replace the iOS 9 check with whether forceTouchCapability is defined by the system OS (regardless of its value?)